### PR TITLE
Add rough draft of AWS info

### PR DESCRIPTION
**Pull request by mjdesa**
Fixes https://github.com/influxdb/influxdb.com/issues/122

### CODE REVIEW

**Comment by mjdesa** [comment]
@beckettsean @gunnaraasen @rkuchan thoughts and opinions would be greatly appreciated.

File: content/docs/v0.9/introduction/installation.md [hunk 1]
```diff
@@ -88,8 +88,81 @@ The output will show every option configured in the `influxdb.partial.conf` file
 
 The example configuration file shipped with the installer is for information only. It is an identical file to the internally generated configuration except that the example file has comments.
 
+## Hosting on AWS
 
+### Hardware
 
+We recommend using an 2 SSD volumes. One for `wal` and one `data`. Depending on your load each volume should have around 1k-3k provisioned IOPS. The `data` volume should have more disk space with lower IOPS and the `wal` volume should have less disk space with higher IOPS.
```
Comment (beckettsean) [inline content/docs/v0.9/introduction/installation.md]: "We recommend using ~~an~~ ~~2~~two SSD volumes, one for the `influxdb/wal` directory and one for the `influxdb/data` directory. 

The `influxdb/wal` volume only stores very recently written data so does not need to be very large, but it should have high IOPS. The `influxdb/data` volume will contain all data but does not need to handle the same IOPS load as `influxdb/wal`.

File: content/docs/v0.9/introduction/installation.md [hunk 2]
```diff
@@ -88,8 +88,81 @@ The output will show every option configured in the `influxdb.partial.conf` file
...
+Each machine should have a minimum of 8G RAM.
+
+We’ve seen the best performance with the C3 class of machines.
+
+## Configuring the Instance
+
+This example assumes that you are using two SSD volumes and that you have mounted them appropriately. This example also assumes that each of those volumes is mounted at `/mnt/influx` and `/mnt/db`. For more infomation on how to do that see the Amazon documentation on how to [Add a Volume to Your Instance](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-add-volume-to-instance.html).
+
+### Config File
+You'll have to update teh config file appropriately for each InfluxDB instance you have.
```
Comment (beckettsean) [inline content/docs/v0.9/introduction/installation.md]: s/teh/the/

**Comment by beckettsean** [comment]
a couple rewords and a few typos, otherwise :+1: 

We will have to reorganize this page soon but I don't have the energy to think about it today.

**Pull request merged**